agent_deploy/kubernetes/sysdig-agent-daemonset-v1.yaml

```diff
@@ -50,6 +50,14 @@ spec:
       - name: varrun-vol
         hostPath:
           path: /var/run
+      - name: podinfo
+        downwardAPI:
+          defaultMode: 420
+          items:
+          - fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
```

Comment:
Should we put the pod name in also since we are changing it?

Reply:
Added. Need new approval